pkg/flowcli/services/transactions.go

```diff
@@ -77,6 +77,31 @@ func (t *Transactions) Send(
 	return t.send(code, signer, args, argsJSON)
 }
 
+// SendForAddressWithCode send transaction for address and private key specified with code
+func (t *Transactions) SendForAddress(
+	transactionFilename string,
+	signerAddress string,
+	signerPrivateKey string,
+	args []string,
+	argsJSON string,
+) (*flow.Transaction, *flow.TransactionResult, error) {
+	code, err := util.LoadFile(transactionFilename)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	address := flow.HexToAddress(signerAddress)
```

Comment:
could you call `SendForAddressWithCode` at this point?

Reply:
Indeed!
Great spot, @janezpodhostnik!